scripts/deploy.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Following assumptions are made by this script:                               #
+# * gcloud, docker, and docker-compose is installed                            #
+# * it is called from $GOPATH/src/github.com/google/keytransparency/scripts    #
```

Comment:
Could we it's assumption to be called from the root directory
` $GOPATH/src/github.com/google/keytransparency` ?